logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationRepository.kt

```diff
@@ -88,7 +88,7 @@ class ConversationDataSource(
 
     override suspend fun fetchConversations(): Either<CoreFailure, Unit> {
         kaliumLogger.d("Fetching conversations")
-        val selfUserTeamId = userRepository.getSelfUser().first().team
+        val selfUserTeamId = userRepository.observeSelfUser().first().teamId
```

Comment:
I would suggest preparing another  suspend func `getSelfUser` to get nullable self user, this will help avoid some functions getting stuck when `observeSelfUser` will filter nullable values

Reply:
Sure